build/Rulesets/Roslyn.ruleset

```diff
@@ -1,115 +1,139 @@
+    <Rule Id="RS0026" Action="Error" />
+    <Rule Id="RS0027" Action="Error" />
+  </Rules>
+  <Rules AnalyzerId="System.Collections.Immutable.Analyzers" RuleNamespace="System.Collections.Immutable.Analyzers">
+    <Rule Id="RS0012" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="System.Runtime.Analyzers" RuleNamespace="System.Runtime.Analyzers">
+    <Rule Id="CA1305" Action="None" />
+    <Rule Id="CA1307" Action="None" />
+    <Rule Id="CA1308" Action="None" />
+    <Rule Id="CA1810" Action="None" />
     <Rule Id="CA1816" Action="None" />
 
-  <!--CA1825 has a bug that fires on compiler generated code and is very noisy in this repo. Reenable this rule when we move to a new version of analyzers.' --> 
-    <Rule Id="CA1825" Action="None" /> 
-    <Rule Id="CA2002" Action="None" /> 
-    <Rule Id="CA2207" Action="None" /> 
-    <Rule Id="CA2208" Action="None" /> 
-    <Rule Id="CA2216" Action="None" /> 
-    <Rule Id="CA2219" Action="None" /> 
-    <Rule Id="CA2241" Action="None" /> 
-    <Rule Id="CA2242" Action="None" /> 
-     <Rule Id="RS0014" Action="Warning" /> 
-   </Rules> 
-   <Rules AnalyzerId="System.Runtime.InteropServices.Analyzers" RuleNamespace="System.Runtime.InteropServices.Analyzers"> 
-     <Rule Id="CA1401" Action="None" /> 
-     <Rule Id="CA2101" Action="None" /> 
-     <Rule Id="RS0015" Action="Warning" /> 
-   </Rules> 
-   <Rules AnalyzerId="System.Threading.Tasks.Analyzers" RuleNamespace="System.Threading.Tasks.Analyzers"> 
-     <Rule Id="RS0018" Action="Warning" /> 
-   </Rules> 
-   <Rules AnalyzerId="XmlDocumentationComments.Analyzers" RuleNamespace="XmlDocumentationComments.Analyzers"> 
-     <Rule Id="RS0010" Action="Warning" /> 
-   </Rules> 
- </RuleSet> 
+    <!--CA1825 has a bug that fires on compiler generated code and is very noisy in this repo. Reenable this rule when we move to a new version of analyzers.' -->
+    <Rule Id="CA1825" Action="None" />
+    <Rule Id="CA2002" Action="None" />
+    <Rule Id="CA2207" Action="None" />
+    <Rule Id="CA2208" Action="None" />
+    <Rule Id="CA2216" Action="None" />
+    <Rule Id="CA2219" Action="None" />
+    <Rule Id="CA2241" Action="None" />
+    <Rule Id="CA2242" Action="None" />
+    <Rule Id="RS0014" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="System.Runtime.InteropServices.Analyzers" RuleNamespace="System.Runtime.InteropServices.Analyzers">
+    <Rule Id="CA1401" Action="None" />
+    <Rule Id="CA2101" Action="None" />
+    <Rule Id="RS0015" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="System.Threading.Tasks.Analyzers" RuleNamespace="System.Threading.Tasks.Analyzers">
+    <Rule Id="RS0018" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="XmlDocumentationComments.Analyzers" RuleNamespace="XmlDocumentationComments.Analyzers">
+    <Rule Id="RS0010" Action="Warning" />
+  </Rules>
+  <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
```

Comment:
I would just remove CSharp from the AnalyzerId and RuleNamespace. Former is used when grouping rules in the ruleset editor. I believe the later is not used at all.